Doc/library/enum.rst

```diff
@@ -504,16 +504,28 @@ Data Types
 
 .. class:: StrEnum
 
-   ``StrEnum`` is the same as :class:`Enum`, but its members are also strings and can be used
-   in most of the same places that a string can be used.  The result of any string
-   operation performed on or with a *StrEnum* member is not part of the enumeration.
+   *StrEnum* is the same as :class:`Enum`, but its members are also strings and
```

Comment:
It looks like the documentation is inconsistent since some chapters use `*StrEnum*` and others use <code>\`\`StrEnum\`\`</code>